extensions/import/src/wizard/pages/summaryPage.ts

```diff
@@ -137,10 +138,14 @@ export class SummaryPage extends ImportPage {
 
 		const currentServer = this.model.server;
 		const includePasswordInConnectionString = (currentServer.options.authenticationType === azdata.connection.AuthenticationType.Integrated) ? false : true;
-		const connectionString = await azdata.connection.getConnectionString(currentServer.connectionId, includePasswordInConnectionString);
+		let connectionString = await azdata.connection.getConnectionString(currentServer.connectionId, includePasswordInConnectionString);
```

Comment:
Is there a way we could fix this call instead to return a valid connection string? Having to have all extensions that do something like this have to implement their own version of the logic in this PR doesn't seem very maintainable.

Reply:
The connection string is valid as per STS - but because this is used "only with access token" with another service - STS cannot be made aware of that, as it decides based on 'mssql:enableSqlAuthenticationProvider' setting. So technically import extension should get connection string from it's own service - but because it gets from STS, it would need to update it's properties.